Makefile

```diff
@@ -89,7 +89,7 @@ container:
 
 	docker build --pull -t $(PREFIX)/heapster-$(ARCH):$(VERSION) $(TEMP_DIR)
 ifneq ($(OVERRIDE_IMAGE_NAME),)
-	docker tag -f $(PREFIX)/heapster-$(ARCH):$(VERSION) $(OVERRIDE_IMAGE_NAME)
+	docker tag $(PREFIX)/heapster-$(ARCH):$(VERSION) $(OVERRIDE_IMAGE_NAME)
```

Comment:
This line never worked for me as https://docs.docker.com/engine/reference/commandline/tag/ has no `-f` flag. I can send a separate PR if you prefer that.